### ISSUE TITLE
Shopping list segmentation fault

### ISSUE THREAD

**Issue by L3K0V** [cooklang/CookCLI#47]
```
Started server on http://127.0.0.1:9080, serving cook files from /Users/azlekov/Workspace/cookbook.
[1]    24627 segmentation fault  cook server
```

Steps to reproduce:
1. `cook seed`
2. `cook server`
3. Click on Shopping list

Steps to reproduce 2:
1. `cook seed`
2. `cook server`
3. Click on recipe and add to shopping list
4. Click on Shopping list

OS: macOS X 12.2.1
Version of the CLI: 0.1.3

**Comment by L3K0V** [comment]
I saw https://github.com/cooklang/CookCLI/issues/44 but any clue when 0.1.4 will be available?

**Comment by dubadub** [comment]
Thanks for submitting the report. Brew package is updated now. The issue fixed in `0.1.4` was related to Linux build only. I was trying to reproduce it on macOS X 12.2.1 and CLI: 0.1.4 but I wasn't able. Was it a specific recipe to add to a shopping list?

**Comment by L3K0V** [comment]
Thanks for publishing the new homebrew tap version! Using 0.1.4 looks to be working.
